lightkurve/periodogram.py

```diff
@@ -581,17 +571,21 @@ def __init__(self, *args, **kwargs):
         self._LS_object = kwargs.pop("ls_obj", None)
         self.nterms = kwargs.pop("nterms", 1)
         self.ls_method = kwargs.pop("ls_method", 'fastchi2')
+        self.normalization = kwargs.pop("normalization")
         super(LombScarglePeriodogram, self).__init__(*args, **kwargs)
 
     def __repr__(self):
-        return('LombScarglePeriodogram(ID: {})'.format(self.label))
+        if self.normalization == 'amplitude':
+            return('Amplitude spectrum (ID: {})'.format(self.label))
+        elif self.normalization == 'psd':
+            return('Power spectral density (ID: {})'.format(self.label))
```

Comment:
Nitpick: I think I'd prefer to keep calling it a `LombScarglePeriodogram` in the `__repr__` to differentiate it from the `BLSPeriodogram`, and because all other Lightkurve classes use the class name in their repr.  However we could highlight the normalization in a different way, e.g.:
```
LombScarglePeriodogram(ID: xxxx, normalization: power spectral density)
```